doc/releases/changelog-dev.md

```diff
@@ -134,6 +134,22 @@
 
   ```
 
+
+* Added new function `qml.bravyi_kitaev` to map fermionic Hamiltonians to qubit Hamiltonians.
+  [(#5390)](https://github.com/PennyLaneAI/pennylane/pull/5390)
+
+```python
+  import pennylane as qml
+  fermi_ham = qml.fermi.FermiWord({(0, 0) : '+', (1, 1) : '-'})
```

Comment:
Not a blocker, but it might be nice to use the `from_string` approach, since we've seen some users unfamiliar with it.